docker/cami-medical-compliance/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM williamyeh/ansible:ubuntu16.04
+
+# or, for example, FROM ansible/ubuntu14.04-ansible:stable
+
+# Add playbooks to the Docker image. This requires that the build context is the
+# project root folder. Run build like:
+# `docker build -f docker/cami-store/Dockerfile .`
+COPY . /cami-project
```

Comment:
It's a reminiscence from how Vagrant did it with it's base folder. A better place might would be /opt/cami-project.